src/Twitter.php

```diff
@@ -803,14 +804,21 @@ public function friendshipsCreate($id, array $params = []) : Response
      *
      * Returns the next cursor if there are more to be returned.
      *
-     * @param int|string $id
+     * $id may be one of any of the following:
+     *
+     * - a single user identifier
+     * - a single screen name
+     * - a list of user identifiers
```

Comment:
I understand the point about parameter widening and child classes. However, an API wrapper like this is less likely to have somebody extend an existing method that maps to the API, and more likely to extend to _add_ methods to interact with API endpoints that have not yet been implemented (e.g., new methods). The primary factor in this case is that the _consumer_ API does not break, versus the _extension_ API.